samples/typescript_nodejs/16.proactive-messages/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "016.proactive-messages",
+    "version": "1.0.0",
+    "description": "Bot Builder v4 proactive messaging sample",
+    "author": "Generated using Microsoft Bot Builder Yeoman generator v4.7.0",
+    "license": "MIT",
+    "main": "./lib/index.js",
+    "scripts": {
+        "build": "tsc --build",
+        "lint": "tslint -c tslint.json 'src/**/*.ts'",
+        "postinstall": "npm run build && node ./deploymentScripts/webConfigPrep.js",
+        "start": "tsc --build && node ./lib/index.js",
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "watch": "nodemon --watch ./src -e ts --exec \"npm run start\""
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com"
+    },
+    "dependencies": {
+        "botbuilder": "~4.7.0",
```

Comment:
Please target 4.8.0 (just released)

Reply:
Yes sorry forgot to do the before doing the PR. Fixed that now